tests/test_indexer.py

```diff
@@ -349,16 +362,16 @@ def test_indexed_file_unparsable(self):
         self.assertRegex(log_monitor.output[0],
                          "WARNING:.*:In bundle .* the file 'unparseable_json.json' is marked for indexing"
                          " yet could not be parsed. This file will not be indexed. Exception:")
-        search_results = self.get_search_results(self.smartseq2_paired_ends_query, 1)
-        self.assertEqual(1, len(search_results))
-        self.verify_index_document_structure_and_content(search_results[0], bundle_key,
-                                                         files=smartseq2_paried_ends_indexed_file_list)
+#        search_results = self.get_search_results(smartseq2_paired_ends_v2_or_v3_query, 1)
```

Comment:
Requested: Delete or comment back in. If the former, could you quickly explain why these assertion aren't needed? In person is fine.

Reply:
deleted

Reply:
![image](https://user-images.githubusercontent.com/5143256/43747368-1dabb2e4-999e-11e8-9e21-6a2b028d1b6d.png)

Reply:
This is after a full page reload.